tests/unit/models/v0.3/modelCard.test.js

```diff
@@ -1081,64 +1050,63 @@ describe("Card", () => {
   });
 
   describe("setExpirationDate", () => {
-    const simplyeCard = new Card({
-      ...basicCard,
-      policy: Policy({ policyType: "simplye" }),
-    });
-    const webCard = new Card({
+    const card = new Card({
       ...basicCard,
       policy: Policy({ policyType: "webApplicant" }),
     });
 
-    it("should set the temporary expiration date to 30 days for simplye policy", () => {
```

Comment:
There's no longer a "standard" and a "temporary" expiration date for a ptype. Each ptype only has one expiration date now.